### ISSUE TITLE
Upgrade jQuery

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/Global-signbank#406]
We are using jQuery 1.11, which is apparently insecure: although I can't find much about jQuery versions and security breaches, for FinSL there has been a notification generated by Github that anything below jQuery 3.0.0 is insecure.

While the upgrade itself will be a matter of replacing a file, I'm not sure what the implications will be for the jQuery code, or for Signbank's dependencies that use jQuery.

**Comment by henrinie** [comment]
That is indeed quite interesting. Apparently there are two potential issues, dating from 2015 and 2016:

* https://nvd.nist.gov/vuln/detail/CVE-2016-10707
* https://nvd.nist.gov/vuln/detail/CVE-2015-9251

It is rather hard to figure out if the jQuery 1 & 2 branches are really affected by this. There seems to be some talk clarifying the 2016 CVE at least https://github.com/jquery/jquery/issues/3133#issuecomment-358978489

It is probably best just to migrate to jQuery 3 to be sure. We switched to jQuery 2 quite a while ago, and if I remember correctly that required some changes, but not that much.
I tried jQuery 3 maybe 3 months ago and had some problems and decided not to make the switch.
